readthedocs/rtd_tests/tests/test_gold.py

```diff
@@ -26,6 +26,11 @@ def test_adding_projects(self):
         self.assertEqual(self.golduser.projects.count(), 1)
         self.assertEqual(resp.status_code, 302)
 
+    def test_incorrect_input_when_adding_projects(self):
+        self.assertEqual(self.golduser.projects.count(), 0)
+        resp = self.client.post(reverse('gold_projects'), data={'project': 'random-incorrect-slug'})
```

Comment:
Would be good to have an assert about that `random-incorrect-slug` doesn't exists on the db